datafusion/common/src/config.rs

```diff
@@ -837,6 +837,18 @@ config_namespace! {
         /// writing out already in-memory data, such as from a cached
         /// data frame.
         pub maximum_buffered_record_batches_per_stream: usize, default = 2
+
+        /// Enable sort pushdown optimization for Parquet files.
+        /// When enabled, optimizes queries with ORDER BY:
+        /// - Reordering files based on statistics
+        /// - Reversing row group read order when beneficial
+        /// Returns **inexact ordering**: Sort operator is kept for correctness,
+        /// but can terminate early for TopK queries (ORDER BY ... LIMIT N),
+        /// providing huge speedup.
+        /// Memory: No additional overhead (only changes read order).
+        /// Future TODO: Will add option to support detecting perfectly sorted data and eliminate Sort completely.
```

Comment:
BTW since parquet files themselves have no way to communicate sort order (I think the pre-existing ordering comes from the LIstingTable level), I would expect the option for eliminating sorts to not be on the parquet options 🤔